src/features/singleRight/components/ResourceActionBar/ResourceActionBar.tsx

```diff
@@ -77,25 +77,25 @@ export const ResourceActionBar = ({
   const addButton = (
     <Button
       variant='tertiary'
-      icon={<PlusCircleIcon title='add' />}
       size={compact ? 'large' : 'medium'}
       onClick={() => {
         onAddClick?.();
       }}
       iconPlacement='right'
     >
+      <PlusCircleIcon title='add' />
       {!compact && (status === ServiceStatus.HTTPError ? t('common.try_again') : t('common.add'))}
     </Button>
   );
 
   const undoButton = (
     <Button
       variant='tertiary'
-      icon={<ArrowUndoIcon title={t('common.undo')} />}
       size={compact ? 'large' : 'medium'}
       onClick={onRemoveClick}
       iconPlacement='right'
     >
+      <ArrowUndoIcon title={t('common.undo')} />
       {!compact && t('common.undo')}
```

Comment:
Should we make this a pattern all places, place the icon on the right?

Reply:
I'm not sure... I noticed in the code that we alternate between having the icons to the left and to the right of the text. It's probably because the design says so, but I'm not sure if it was a conscious decision there either… 😅

Reply:
Maybe that's good thing for you to discuss with UX to establish some patterns? 😁